### PR TITLE
fix(data-service-gen): Standardize code generation output

### PR DESCRIPTION
**Pull request by Sadeeed**
Close: #7049

## PR Details

Standardize the code generation logs according to the format described in the issue mentioned above.

### Summary
:pencil2::recycle::books:

Refactored logger messages in `data-service-generator` package to improve readability and consistency. Fixed spelling, capitalization, and grammar errors in logger messages and filenames.

> _When creating a server with `create-server.ts`_
> _The logger messages were a bit of a mess_
> _They used wrong spelling and tense_
> _And lacked proper sense_
> _So they were improved with some finesse_

### Walkthrough
*  Fix spelling and capitalization of Dotenv and Docker Compose in logger messages ([link](https://github.com/amplication/amplication/pull/7088/files?diff=unified&w=0#diff-7e369d3fdc3528cd25d13aadc06d5132caf0697deaf539d51f7bbc36f53099d6L115-R115), [link](https://github.com/amplication/amplication/pull/7088/files?diff=unified&w=0#diff-9c1a5a055f94dfa5f9906d14a9dc82594d750eaa22bd3878db57d21e96f62aaaL105-R105), [link](https://github.com/amplication/amplication/pull/7088/files?diff=unified&w=0#diff-9c1a5a055f94dfa5f9906d14a9dc82594d750eaa22bd3878db57d21e96f62aaaL113-R113), [link](https://github.com/amplication/amplication/pull/7088/files?diff=unified&w=0#diff-3f2a063de14588fbf23cc6430e18d40f7943f12c78006d20f21a86cd745323fdL181-R181))
*  Use gerund form of creating in logger messages for consistency and clarity ([link](https://github.com/amplication/amplication/pull/7088/files?diff=unified&w=0#diff-1a11956a0b46f9708ce929f09590637ce92db2e8ac50a992de77c6c4ddf6ad9cL18-R21), [link](https://github.com/amplication/amplication/pull/7088/files?diff=unified&w=0#diff-20258b9ec6522f213cae68ed72603771aeb86e859bee4ef00e3f10cb922cc573L12-R12), [link](https://github.com/amplication/amplication/pull/7088/files?diff=unified&w=0#diff-9c1a5a055f94dfa5f9906d14a9dc82594d750eaa22bd3878db57d21e96f62aaaL85-R85), [link](https://github.com/amplication/amplication/pull/7088/files?diff=unified&w=0#diff-c353c879311b8efa11292a74a97b12d097cf7c9d3f4947ba02cf5baf76983bd7L19-R25))
*  Use acronym DTOs instead of dtos in logger message for readability and convention ([link](https://github.com/amplication/amplication/pull/7088/files?diff=unified&w=0#diff-9c1a5a055f94dfa5f9906d14a9dc82594d750eaa22bd3878db57d21e96f62aaaL85-R85))

## PR Checklist

- [ ] Tests for the changes have been added
- [ ] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

File: packages/data-service-generator/src/server/create-types-related-files/create-types-related-files.ts [hunk 1]
```diff
@@ -16,13 +16,13 @@ export async function createTypesRelatedFiles(): Promise<ModuleMap> {
 
   const moduleMap = new ModuleMap(logger);
 
-  await logger.info("Create GraphqlBitIntFile file...");
+  await logger.info("Creating GraphqlBitIntFile...");
```
Comment (yuval-hazaz) [inline packages/data-service-generator/src/server/create-types-related-files/create-types-related-files.ts:19]:
```suggestion
  await logger.info("Creating GraphqlBitInt file...");
```

**Review by yuval-hazaz — CHANGES_REQUESTED** [review]
hi @Sushrut1101 thank you for your contribution.
please see a minor comment

File: packages/data-service-generator/src/server/create-types-related-files/create-types-related-files.ts (hunk 1 above)
Comment (Sadeeed) [reply packages/data-service-generator/src/server/create-types-related-files/create-types-related-files.ts]: Resolved 🙂

**Review by yuval-hazaz — APPROVED** [review]

**Comment by yuval-hazaz** [comment]
Thanks and congrats on your first PR with Amplication 🔥

**Comment by Sadeeed** [comment]
> Thanks and congrats on your first PR with Amplication 🔥

Thankyou 🙇

**Pull request merged**